python_modules/libraries/dagster-dbt/dagster_dbt/include/setup.py.jinja

```diff
@@ -4,7 +4,7 @@ setup(
     name="{{ project_name }}",
     version="0.0.1",
     packages=find_packages(),
-    {%- if use_dbt_project_package_data_dir %}
+    {%- if use_experimental_dbt_project %}
```

Comment:
[2] the main downside in my view of merging the flags and having this stuff always on is still remains. We will end up doing the directory sync in prepare and having the setup.py refer to package data that an outside of serverless opt in user likely does not need. 

That said, its not clear how material that downside is in practice.